### PR TITLE
Migration: winforms & wpa group updates

### PR DESCRIPTION
**Pull request by guardrex**
ready for :eyes:

### CODE REVIEW

File: docs/framework/configure-apps/file-schema/winforms/windows-forms-add-configuration-element.md
Comment (mairaw) [inline docs/framework/configure-apps/file-schema/winforms/windows-forms-add-configuration-element.md]: i've noticed this topic and some others are missing the ms.prod value which is a required metadata. we should file an issue to take care of that.

File: docs/framework/winforms/high-dpi-support-in-windows-forms.md
Comment (mairaw) [inline docs/framework/winforms/high-dpi-support-in-windows-forms.md]: your application seemed correct here

File: docs/framework/wpf/advanced/the-ink-object-model-windows-forms-and-com-versus-wpf.md
Comment (mairaw) [inline docs/framework/wpf/advanced/the-ink-object-model-windows-forms-and-com-versus-wpf.md]: wondering if we should perhaps add this api to the https://github.com/dotnet/docs/blob/master/_zip/missingapi.yml file. Or at least I'd link to version that is linked to the TOC: https://msdn.microsoft.com/en-us/library/microsoft.ink.ink.load(v=vs.90).aspx

File: docs/framework/configure-apps/file-schema/winforms/windows-forms-add-configuration-element.md
Comment (GuardRex) [inline docs/framework/configure-apps/file-schema/winforms/windows-forms-add-configuration-element.md]: https://github.com/dotnet/docs/issues/2061

File: docs/framework/winforms/high-dpi-support-in-windows-forms.md
Comment (GuardRex) [inline docs/framework/winforms/high-dpi-support-in-windows-forms.md]: yep ... typo ... thx

File: docs/framework/wpf/advanced/the-ink-object-model-windows-forms-and-com-versus-wpf.md
Comment (GuardRex) [inline docs/framework/wpf/advanced/the-ink-object-model-windows-forms-and-com-versus-wpf.md]: Link updated. Noted API in Migration Project for additional work.

**Review by rpetrusha — CHANGES_REQUESTED** [review]
@Guardrex, can you add the top-level file-schema/winforms/index.md to your PR? The syntax section for the `ConfigurationSection` element has `\<` for both the begin and end tags. Also, the first entry int he "See also" section needs to be terminated by 3 spaces. Also, the element name is System.Windows.Forms.ApplicationConfigurationSection.

**Comment by GuardRex** [comment]
@rpetrusha 

> add the top-level file-schema/winforms/index.md

... in *docs\framework\configure-apps\toc.md*? If so, then updates are complete.

File: docs/framework/configure-apps/file-schema/winforms/windows-forms-add-configuration-element.md
Comment (rpetrusha) [inline docs/framework/configure-apps/file-schema/winforms/windows-forms-add-configuration-element.md]: The original is correct in both cases (ApplicationConfigurationSection, not ConfigurationSection)

File: docs/framework/winforms/high-dpi-support-in-windows-forms.md
Comment (rpetrusha) [inline docs/framework/winforms/high-dpi-support-in-windows-forms.md]: ApplicationConfigurationSection is correct.

File: docs/framework/winforms/high-dpi-support-in-windows-forms.md
Comment (rpetrusha) [inline docs/framework/winforms/high-dpi-support-in-windows-forms.md]: `ApplicationConfigurationSection ` is correct in both cases.

File: docs/framework/winforms/high-dpi-support-in-windows-forms.md
Comment (rpetrusha) [inline docs/framework/winforms/high-dpi-support-in-windows-forms.md]: The original `ApplicationConfigurationSection` is correct in both cases.

**Review by rpetrusha — CHANGES_REQUESTED** [review]
The major issue here is that the documentation was inconsistent in the name of the config element. The correct name is `System.Windows.Forms.ApplicationConfigurationSection`.

**Comment by GuardRex** [comment]
@rpetrusha Updates complete.

File: docs/framework/configure-apps/file-schema/winforms/index.md
Comment (rpetrusha) [inline docs/framework/configure-apps/file-schema/winforms/index.md]: ApplicationConfigurationSection for both line 23 and line 25

**Review by rpetrusha — COMMENTED** [review]
Sorry, @GuardRex, I missed one ConfigurationSection -> ApplicationConfigurationSection.

File: docs/framework/configure-apps/file-schema/winforms/index.md
Comment (GuardRex) [inline docs/framework/configure-apps/file-schema/winforms/index.md]: Lines 17, 51, and 53 look like they need this change, too.

Comment (GuardRex) [inline docs/framework/configure-apps/file-schema/winforms/index.md]: ... and same thing with lines 68 and 70 in *windows-forms-add-configuration-element.md*.

File: docs/framework/configure-apps/toc.md
Comment (rpetrusha) [inline docs/framework/configure-apps/toc.md]: This should be removed -- Windows Forms configuration section is a child of Configuration file schema, and it's already listed in the Configuration file schema topic.

Comment (rpetrusha) [inline docs/framework/configure-apps/toc.md]: This should be removed. The topic is a parent of the Configuration file schema topic, and it already appears both in the TOC and as a link from that topic.

**Review by rpetrusha — APPROVED** [review]
@GuardRex, aside from my comment about the TOC entry, this looks good. After you address the TOC issue, you can merge this when the build completes.

**Pull request merged**
